projects.json

```diff
@@ -7,6 +7,7 @@
         "https://github.com/ubiquity/scraper-kernel",
         "https://github.com/ubiquity/bounty-bot",
         "https://github.com/ubiquity/ubiquity-dollar",
-        "https://github.com/ubiquity/devpool-directory-bounties"
+        "https://github.com/ubiquity/devpool-directory-bounties",
+        "https://github.com/harjaapdhillon16/devpool-directory-bounty"
```

Comment:
Why did you add your own repo? Pls remove it

Reply:
done